lib/python/qmk/cli/mass_compile.py

```diff
@@ -23,7 +23,7 @@ def mass_compile_targets(targets: List[BuildTarget], clean: bool, dry_run: bool,
     os.environ.setdefault('SKIP_SCHEMA_VALIDATION', '1')
 
     make_cmd = find_make()
-    builddir = Path(QMK_FIRMWARE) / '.build'
+    builddir = Path(env.get("BUILD_DIR")) if env.get("BUILD_DIR") is not None else Path(QMK_FIRMWARE) / '.build'
```

Comment:
Could be simplified to `Path(env.get('BUILD_DIR') or QMK_FIRMWARE)` / '.build'`

Reply:
The convention is not to append .build to BUILD_DIR I believe, maybe I can do
```python
Path(env.get('BUILD_DIR') or f"{QMK_FIRMWARE}/.build")
```
or
```python
Path(env.get('BUILD_DIR', f"{QMK_FIRMWARE}/.build"))
```
I was fearing this would be too much of a change.
let me know if it's acceptable.